### ISSUE TITLE
Crash  NoSuchMethodError: No virtual method buildUpon()

### ISSUE THREAD

**Issue by fullflash** [ened/flutter_plugin_media_info#25]
```
selector.setParameters(
          selector.getParameters().buildUpon().setRendererDisabled(indexOfAudioRenderer, true));
```

```
ava.lang.NoSuchMethodError: 
No virtual method buildUpon()Lcom/google/android/exoplayer2/trackselection/DefaultTrackSelector$ParametersBuilder;
in class Lcom/google/android/exoplayer2/trackselection/DefaultTrackSelector$Parameters; or its super classes (declaration of 'com.google.android.exoplayer2.trackselection.DefaultTrackSelector$Parameters' appears in /data/app/com.bazzptt.bazz_flutter-WdeANXRggyGjMYcMGAqUvg==/base.apk)
[        ] E/AndroidRuntime( 5793): 	at asia.ivity.mediainfo.MediaInfoPlugin.ensureExoPlayer(MediaInfoPlugin.java:400)
[        ] E/AndroidRuntime( 5793): 	at asia.ivity.mediainfo.MediaInfoPlugin.handleMediaInfoExoPlayer(MediaInfoPlugin.java:176)
```

**Comment by kjawadDeveloper2** [comment]
@ened  do you have any update on this.

**Comment by ened** [comment]
The exoplayer version resolved in your project must match media_info's expectation. 

I'm afraid you had a mismatch here. Please do take a look at the linked PR and also run `gradle app:dependencies` to ensure the right exoplayer version is found.

**Comment by ened** [comment]
Fixed in #26
